iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/SubstationImpl.java

```diff
@@ -155,7 +155,7 @@ public int getThreeWindingsTransformerCount() {
     @Override
     public Substation addGeographicalTag(String tag) {
         if (tag == null) {
-            throw new ValidationException(this, "geographical tag is null");
+            throw new IllegalArgumentException("geographical tag is null");
```

Comment:
Please put this implementation (without the listeners) as default implementation in the interface and delete it here (same for `getGeographicalTags()`)